frame/staking/src/benchmarking.rs

```diff
@@ -494,6 +517,36 @@ mod tests {
 		});
 	}
 
+	#[test]
+	fn add_slashing_spans_works() {
+		ExtBuilder::default().has_stakers(false).build().execute_with(|| {
+			let n = 10;
+
+			let validator_stash = create_validator_with_nominators::<Test>(
+				n,
+				<Test as Trait>::MaxNominatorRewardedPerValidator::get() as u32,
+			).unwrap();
+
+			// Add 20 slashing spans
+			let num_of_slashing_spans = 20;
+			add_slashing_spans::<Test>(&validator_stash, num_of_slashing_spans);
+
+			let slashing_spans = SlashingSpans::<Test>::get(&validator_stash).unwrap();
+			assert_eq!(slashing_spans.iter().count(), num_of_slashing_spans as usize);
+			for i in 0 .. num_of_slashing_spans {
+				assert!(SpanSlash::<Test>::contains_key((&validator_stash, i)));
+			}
+
+			// Test everything is cleaned up
+			assert_ok!(Staking::kill_stash(&validator_stash, 20));
```

Comment:
nit: why pass `20`explicitly and not `num_of_slashing_spans`?
```suggestion
			assert_ok!(Staking::kill_stash(&validator_stash, num_of_slashing_spans));
```